Functions/xASL_Iteration.m

```diff
@@ -328,11 +328,18 @@
         end
         
         % Some feedback about this iteration (after opening diary log)
+                
         if ~AlreadyProcessed
             fprintf('\n%s\n',repmat('+',1,72)); % just draw a separator line
-            fprintf('%s\n',['Starting ' x.MUTEXID ' at ' datestr(now)]);
+            [StartI, EndI] = regexp(diaryFileEx, 'ASL_\d','ONCE'); %to find the name of the session inside diaryFileEx: ASL_with any digit after
+            session= diaryFileEx(StartI:EndI); %isolate "ASL_1 or ASL_2 etc
+            if ~isempty (session) %writes the session only for ASL module
```

Comment:
Nice! You're sure that `session`exists for other modules? Otherwise `~isempty(session)` would crash